lib/ruby/stdlib/rubygems/command_manager.rb

```diff
@@ -169,12 +169,6 @@ def process_args(args, build_args=nil)
     when '-v', '--version' then
       say Gem::VERSION
       terminate_interaction 0
-    when '--no-ri', '--no-rdoc' then
```

Comment:
Minor concern here as these flags will now error.

Reply:
Actually nevermind... they errored before. The difference is that they use the deprecation mechanism instead of having special code.